openmc/mgxs_library.py

```diff
@@ -1086,12 +1092,13 @@ def set_nu_fission_mgxs(self, nu_fission, temperature=294.,
     def set_prompt_nu_fission_mgxs(self, prompt_nu_fission, temperature=294.,
                                    nuclide='total', xs_type='macro',
                                    subdomain=None):
-        """This method allows for an openmc.mgxs.PromptNuFissionXS to be used to
-        set the prompt-nu-fission cross section for this XSdata object.
+        """This method allows for an openmc.mgxs.PromptNuFissionXS or
+        openmc.mgxs.PromptNuFissionMatrixXS to be used to set the
+        prompt-nu-fission cross section for this XSdata object.
```

Comment:
Docstrings should have a one-line summary followed by an optional paragraph that elaborates.